osf/management/commands/file_view_counts.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import unicode_literals
```

Comment:
This file should be named something more descriptive. Something lke `migrate_file_view_counts_from_keen.py`